101-hdinsight-linux-ssh-publickey-metastore-vnet/azuredeploy.json

```diff
@@ -149,7 +148,7 @@
             },
             "hive-site": {
               "javax.jdo.option.ConnectionDriverName": "com.microsoft.sqlserver.jdbc.SQLServerDriver",
-              "javax.jdo.option.ConnectionURL": "[concat('jdbc:sqlserver://', reference(resourceId(parameters('existingClusterStorageResourceGroup'), 'Microsoft.Sql/servers', parameters('existingHiveMetastoreServerName')), '2015-05-01-preview').fullyQualifiedDomainName, ';database=',parameters('existingHiveMetastoreDatabaseName'),';encrypt=true;trustServerCertificate=true;create=false;loginTimeout=300')]",
+              "javax.jdo.option.ConnectionURL": "[concat('jdbc:sqlserver://', reference(resourceId('Microsoft.Sql/servers',parameters('existingHiveMetastoreServerName')), '2018-06-01-preview').fullyQualifiedDomainName, ';database=',parameters('existingHiveMetastoreDatabaseName'),';encrypt=true;trustServerCertificate=true;create=false;loginTimeout=300')]",
```

Comment:
need to include the resourceGroupName parameter in the resourceId function - otherwise this will fail if the SQL Server is in a separate resourceGroup